Update colors for Demo Architecture to match website theme #5360

olamideTiana opened the pull request:
done

olamideTiana commented:
@svrnm I have been able to recover the file and replace with the original file by using git log to find the commit hash where the original version of the file exists, restore the file and commit the restoration

olamideTiana commented:
@svrnm I updated the colors and replaced it to gray in dark mode and different text colour. Please, let me know your feedback boss, thank you

svrnm commented on layouts/partials/ecosystem/registry/entry.html:
```diff
@@ -141,6 +141,7 @@ <h4 class="alert-heading">Deprecated</h4>
                     {{ end -}}
                     {{- .description | markdownify -}}
                 </div>
+
```
remove that extra line and it should disappear from the PR

olamideTiana replied:
okay

olamideTiana commented:
> @svrnm I updated the colors and replaced it to gray in dark mode and different text colour. Please, let me know your feedback boss, thank you

Please, let me know your thoughts on this, thank you

svrnm commented on content/en/docs/demo/architecture.md:
```diff
@@ -59,18 +59,18 @@ shippingservice -->|HTTP| quoteservice
-classDef ruby fill:#701516,color:white;
-classDef rust fill:#dea584,color:black;
-classDef typescript fill:#e98516,color:black;
+classDef dotnet fill:#1E1E1E,color:#61AFEF;
+classDef cpp fill:#1E1E1E,color:#C678DD;
+classDef erlang fill:#1E1E1E,color:#E06C75;
+classDef golang fill:#1E1E1E,color:#98C379;
+classDef java fill:#1E1E1E,color:#E5C07B;
+classDef javascript fill:#1E1E1E,color:#E5C07B;
+classDef kotlin fill:#1E1E1E,color:#C678DD;
+classDef php fill:#1E1E1E,color:#61AFEF;
+classDef python fill:#1E1E1E,color:#FFB86C;
+classDef ruby fill:#1E1E1E,color:#E06C75;
+classDef rust fill:#1E1E1E,color:#F8F8F2;
+classDef typescript fill:#1E1E1E,color:#61AFEF;
```
While it is visually much better in that way of having a gray background and a colored text, it's a lot harder to match the boxes at the bottom (that represent the programming language used) with the boxes in the graph, especially also from an accessibility standpoint. So, can you take colors from existing visualizations for the background and then based on the color of the background make the text either black or white.

Here are some other visualizations that we have:

https://deploy-preview-5467--opentelemetry.netlify.app/img/waterfall-trace.svg
https://deploy-preview-5467--opentelemetry.netlify.app/docs/concepts/instrumentation-scope/spans-with-instrumentation-scope.svg
https://deploy-preview-5467--opentelemetry.netlify.app/docs/collector/img/otel-collector.svg

mviitane reviewed (commented):
Unfortunately, I feel the color differences in the arch diagram are now too small. It's very difficult to differentiate for example .NET, PHP and TypeScript from each other.

![image](https://github.com/user-attachments/assets/ef176c22-b80f-470b-9d6f-3a317147e4c0)

mviitane commented:
What about adding the language to the arch diagram as text?
![image](https://github.com/user-attachments/assets/d4a8a533-3d3e-430c-b347-11ef2f9fe4df)

Then we wouldn't even need the "Service legend" pic.

olamideTiana commented:
> What about adding the language to the arch diagram as text? ![image](https://private-user-images.githubusercontent.com/74129181/379696136-d4a8a533-3d3e-430c-b347-11ef2f9fe4df.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk3NzkxNjcsIm5iZiI6MTcyOTc3ODg2NywicGF0aCI6Ii83NDEyOTE4MS8zNzk2OTYxMzYtZDRhOGE1MzMtM2QzZS00MzBjLWIzNDctMTFlZjJmOWZlNGRmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDI0VDE0MDc0N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg0YjVmMmU3MDM2ODA3MzVjYjQyMDgzZGIwZjg0MWFiNmU4NjhhMDNiZjg1ZTVjNjFjOTA3NjlhNmM0Nzc3MmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.d-OcyD8H0asY8_l-HRwgYqsSBtaYOyl6taL5GwtWClY)
> 
> Then we wouldn't even need the "Service legend" pic.

Okay, let me try that. Thanks

svrnm commented:
@olamideTiana are you still interested in helping with that? TY

svrnm commented:
@olamideTiana are you still interested in helping with that? TY

olamideTiana commented:
@svrnm Yes, I'll be glad to help, sorry I've been offline... I'll get right on it

svrnm commented:
> @svrnm Yes, I'll be glad to help, sorry I've been offline... I'll get right on it

Awesome, if necessary, please close this PR and raise a new one since your code might be out of sync

svrnm commented:
@olamideTiana I will close this PR now, please raise a new one based on latest repository content if you still want to work on this
